### PR TITLE
Pass seeking cost function

### PR DESCRIPTION
**Pull request by Yudai-8-om**
## Description
Add cost if:
- Opponents block the way (+999)
- Recipient is surrounded by opponents
- We pass toward downfield

Weight and normalization should be tweaked.

### CODE REVIEW

**Comment by kfu02** [comment]
I'm going to hold off on reviewing this until #1644 is merged and you can actually test this cost function. Should work out of the box with pass_tactic.

File: rj_gameplay/stp/passing_cost.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		punish_amplifier = 1
+
+		our_robots = world_state.our_robots
+		their_robots = world_state.their_robots
+		recipient = robot
+		kicker = [robot for robot in our_robots if robot.has_ball_sense]
+		try:
+			kicker = kicker[0]
+		except:
+			print("Our team is not possessing the ball.")
```
Comment (HussainGynai) [inline rj_gameplay/stp/passing_cost.py]: you should probably return a float('inf') if we don't posses the ball (this is a float which acts as infinity).

Comment (kfu02) [reply rj_gameplay/stp/passing_cost.py]: For the wall_tactic, I got a ValueError when returning float('inf') for invisible robots. @Yudai-8-om you might have to change this back to `return 9999` or something similar. Once you can actually test this cost func this should be more obvious.

**Review by HussainGynai — APPROVED** [review]
LGTM

**Pull request closed without merging**